torch_xla/csrc/tensor.h

```diff
@@ -1248,14 +1249,20 @@ class XLATensor : public c10::intrusive_ptr_target {
     size_t emitted_nodes = 0;
     std::shared_ptr<xla::ComputationClient::Computation> computation;
     std::vector<torch::lazy::BackendDataPtr> parameters_data;
+    bool is_sharded = false;
   };
 
   struct CachedComputation {
     CachedComputation(
         std::shared_ptr<xla::ComputationClient::Computation> computation)
         : computation(std::move(computation)) {}
+    CachedComputation(
```

Comment:
can we just use the old constructor and make `is_sharded` default to false?

Reply:
Good point, thanks -- done.